core/src/trezor/ui/confirm.py

```diff
@@ -225,26 +231,33 @@ def create_tasks(self) -> Tuple[loop.Task, ...]:
 
 
 class HoldToConfirm(ui.Layout):
-    DEFAULT_CONFIRM = "Hold To Confirm"
+    DEFAULT_CONFIRM = "Hold to confirm"
     DEFAULT_CONFIRM_STYLE = ButtonConfirm
     DEFAULT_LOADER_STYLE = LoaderDefault
+    DEFAULT_CANCEL = res.load(ui.ICON_CANCEL)
+    DEFAULT_CANCEL_STYLE = ButtonAbort
 
     def __init__(
         self,
         content: ui.Component,
         confirm: str = DEFAULT_CONFIRM,
         confirm_style: ButtonStyleType = DEFAULT_CONFIRM_STYLE,
         loader_style: LoaderStyleType = DEFAULT_LOADER_STYLE,
+        cancel: Optional[ButtonContent] = DEFAULT_CANCEL,
```

Comment:
is this really `Optional`? your code doesn't seem to support setting `None` here.

(I suggest removing the `Optional`, to do it properly you'd probably want to have full-width hold-to-confirm if this is `None`, and shortened otherwise?)

also I'm not sure how much of a good idea providing this really is, perhaps `DEFAULT_CANCEL` should be used always for `HoldToConfirm` ?

Reply:
Good point, I think it is best to remove it completely.

Reply:
i would keep `cancel_style` though, that seems possibly useful

Reply:
hah, i see i'm too late :eyes: